qa/tasks/vstart_runner.py

```diff
@@ -307,7 +307,7 @@ def _perform_checks_and_return_list_of_args(self, args, omit_sudo):
 
     def run(self, args, check_status=True, wait=True,
             stdout=None, stderr=None, cwd=None, stdin=None,
-            logger=None, label=None, env=None, timeout=None, omit_sudo=True):
+            logger=None, label=None, env=None, timeout=None, omit_sudo=False):
```

Comment:
Because nobody would use `sudo`and intend to get rid of it from the command arguments before it's execution. Also, teuthology's run already has `omit_sudo` set to `False`. So this should be safe. Teuthology's run - https://github.com/ceph/teuthology/blob/master/teuthology/orchestra/run.py#L381

Reply:
Fair enough. Maybe we should just get rid of that parameter altogether? It doesn't seem to make a lot of sense, but maybe I'm missing something.

Reply:
Yes. Now that I am well-acquainted with vstart_runner.py, I think the same.

When I added that parameter, there was a comment that said "we don't need stinkin' sudo" due to which I wasn't sure why not. I'll do this on a different PR.